README.md

```diff
@@ -1 +1,106 @@
+
+- **Pod Network Corruption**: Inject network packet corruption into application pod. Visit <a href="https://docs.litmuschaos.io/docs/pod-network-corruption"> pod network corruption docs</a> for more info.
+
+- **Pod Network Latency**: This experiment causes flaky access to application replica by injecting network delay using pumba. It injects latency on the specified container by starting a traffic control (tc) process with netem rules to add egress delays. It Can test the application's resilience to lossy/flaky network. Visit pod <a href="https://docs.litmuschaos.io/docs/pod-network-latency"> network latency docs</a> 
+
+- **Pod Network Loss**: This experiment injects chaos to disrupt network connectivity to kubernetes pods.The application pod should be healthy once chaos is stopped. It causes loss of access to application replica by injecting packet loss using pumba. Visit <a href="https://docs.litmuschaos.io/docs/pod-network-loss"> pod network loss docs</a>
+
+## Run a chaos experiment using this action
+
+We just need  to follow these simple steps to run a chaos experiment using this action:
+
+- **Install Litmus**: Before running any experiment we need to setup litmus in the cluster. If litmus is not already installed then we can install it from action by just passing an evn `INSTALL-LITMUS` with `true` value or else we can keep it `false`. You can find more details under `setup` folder.
+
+- **Deploy Application**: After having litmus installed in the cluster we need to have an application runnning on which the chaos will be performed. We can create the application also by passing `APP_DEPLOY` as `true` in env along with other variables using this action. You can find more details under `setup` folder.
+
+- **Select experiment**: Select an experiment from experiment folder which you want to use. Get the details of how to run the experiment there only. A sample pod delte experiment workflow is shown here.
+
+
+
+## Usage
+
+`.github/workflows/main.yml`
+
+```yaml
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+      
+    - name: Running Litmus pod delete chaos experiment
+      uses: mayadata-io/github-choas-actions@master
+      env:
+        KUBE_CONFIG_DATA: ${{ secrets.KUBE_CONFIG_DATA }}
+        ##If litmus is not installed
+        INSTALL_LITMUS: true
+        ##Give application info under chaos
+        APP_NS: default
+        APP_LABEL: run=nginx
+        APP_KIND: deployment
+        EXPERIMENT_NAME: pod-delete
+        ##Custom image can also been used
+        EXPERIMENT_IMAGE: litmuschaos/ansible-runner:latest        
+        TOTAL_CHAOS_DURATION: 30
+        CHAOS_INTERVAL: 10
+        FORCE: false
+        ##Select true if you want to uninstall litmus after chaos
+        LITMUS_CLEANUP: true
+```
+
+## Secrets
```

Comment:
Can we place this section in a pre-requisites file and link from here ? README should be purely steps/action - and these preparatory/auxiliary info should be separated out.